### PR TITLE
Feature interfaz

### PR DESCRIPTION
**Pull request by Paola-Xaq**

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/view/MainInterface.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package org.fundacion.jala.converter.view;
+
+
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/view/MainInterface.java]: Delete blank line

File: src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        thumbnailOption = new JCheckBox("With Thumbnail");
+        thumbnailOption.setFont(new Font("Barlow", 0, 12));
+        setLayout(new GridLayout(3, 2));
+        add(resolutionLabel.getTextLabel());
+        add(resolutionSelect);
+        add(frameLabel.getTextLabel());
+        add(framesSelect);
+        add(optionCSound);
+        add(thumbnailOption);
+
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java]: Delete blank line

**Review by JRojas999 — APPROVED** [review]

**Review by JorgeRCV — APPROVED** [review]

**Review by jpgonzales27 — APPROVED** [review]

**Review by GustavoZHA — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/view/metadata/SelectFile.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    protected SelectFile() {
+        selectFile = new JButton("Open File");
+        selectFile.addActionListener(this);
+        int textFieldHeight = 30;
+        int textFieldWidth = 350;
+        textFile = new JTextField();
+        textFile.setPreferredSize(new Dimension(textFieldWidth, textFieldHeight));
+        textFile.setEnabled(false);
+
+        int marginHeight = 30;
```
Comment (SaulCM) [inline src/main/java/org/fundacion/jala/converter/view/metadata/SelectFile.java]: Magic numbers, could be set as final attributes.

**Review by SaulCM — APPROVED** [review]
Seems good.

**Review by JessickaMoya — APPROVED** [review]

**Review by Raymundogs — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/view/MainInterface.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        JTabbedPane mainTabPanel = new JTabbedPane();
+        mainTabPanel.add("Compiler", compilerJP);
+        mainTabPanel.add("Video Converter", video);
+        mainTabPanel.add("Audio Converter", audio);
+        mainTabPanel.add("Meta Data", metaData);
+
+        setLayout(new BorderLayout());
+        add(titleTxt.getTextLabel(), BorderLayout.NORTH);
+        add(mainTabPanel, BorderLayout.CENTER);
+
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/view/MainInterface.java]: these spaces in the code should be removed

File: src/main/java/org/fundacion/jala/converter/view/compiler/CodeTextArea.java [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+        lineCode.setBorder(new EmptyBorder(5, 10, 0, 10));
+        lineCode.setBackground(Color.darkGray);
+        lineCode.setForeground(Color.white);
+
+        JScrollPane textCodeArea = new JScrollPane();
+        textCodeArea.setBorder(new EmptyBorder(0, 0, 0, 0));
+
+        codeArea.getDocument().addDocumentListener(new DocumentListener() {
+            public String getText() {
+                int caretPosition = codeArea.getDocument().getLength();
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/view/compiler/CodeTextArea.java]: these spaces in the code should be removed

File: src/main/java/org/fundacion/jala/converter/view/compiler/CompilerButtons.java [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+        clearConsole = new JButton("Clear");
+        clearConsole.setPreferredSize(new Dimension(80, 20));
+        clearConsole.setFont(new Font("Barlow", 0,  11));
+
+        selectFile = new JButton("Open File");
+        selectFile.setPreferredSize(new Dimension(80, 20));
+        selectFile.setFont(new Font("Barlow", 0,  11));
+        selectFile.addActionListener(this);
+
+        setLayout(new FlowLayout(FlowLayout.RIGHT, 15, 15));
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/view/compiler/CompilerButtons.java]: these spaces in the code should be removed

File: src/main/java/org/fundacion/jala/converter/view/compiler/CompilerInterface.java [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+        panelConstraint.gridx = 3;
+        panelConstraint.gridy = 7;
+        panelConstraint.gridheight = 1;
+        panelConstraint.gridwidth = 2;
+        panelConstraint.weighty = 0;
+        add(buttonsCompiler, panelConstraint);
+
+        panelConstraint.gridx = 1;
+        panelConstraint.gridy = 8;
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/view/compiler/CompilerInterface.java]: these spaces in the code should be removed

**Review by danielaSantaCruz — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/view/compiler/CompilerInterface.java (hunk 7 above)
Comment (Paola-Xaq) [reply src/main/java/org/fundacion/jala/converter/view/compiler/CompilerInterface.java]: Solved, thanks for comment.

File: src/main/java/org/fundacion/jala/converter/view/compiler/CompilerButtons.java (hunk 6 above)
Comment (Paola-Xaq) [reply src/main/java/org/fundacion/jala/converter/view/compiler/CompilerButtons.java]: Solved. :)

File: src/main/java/org/fundacion/jala/converter/view/compiler/CodeTextArea.java (hunk 5 above)
Comment (Paola-Xaq) [reply src/main/java/org/fundacion/jala/converter/view/compiler/CodeTextArea.java]: Solved 👍

File: src/main/java/org/fundacion/jala/converter/view/MainInterface.java (hunk 4 above)
Comment (Paola-Xaq) [reply src/main/java/org/fundacion/jala/converter/view/MainInterface.java]: Solved 👍

File: src/main/java/org/fundacion/jala/converter/view/metadata/SelectFile.java (hunk 3 above)
Comment (Paola-Xaq) [reply src/main/java/org/fundacion/jala/converter/view/metadata/SelectFile.java]: Thanks for suggestion, it is solved now :)

File: src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java (hunk 2 above)
Comment (Paola-Xaq) [reply src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java]: Thanks for noticing, solved :)

File: src/main/java/org/fundacion/jala/converter/view/MainInterface.java (hunk 1 above)
Comment (Paola-Xaq) [reply src/main/java/org/fundacion/jala/converter/view/MainInterface.java]: Solved :)

**Review by cristianfchq — APPROVED** [review]

**Review by edsonarios — APPROVED** [review]
Good job

**Pull request merged**